content/solr/security/2021-12-10-cve-2021-44228.md

```diff
@@ -11,30 +11,21 @@ Critical
 **Description:**
 Apache Solr releases prior to 8.11.1 were using a bundled version of the Apache Log4J library vulnerable to RCE. For full impact and additional detail consult the Log4J security page.
 
-Apache Solr releases prior to 7.4 (i.e. Solr 5, Solr 6, and Solr 7 through 7.3) use log4j 1.2.17 which may be vulnerable for installations using non-default logging configurations that include the JMS Appender, see <https://github.com/apache/logging-log4j2/pull/608#issuecomment-990494126> for discussion.
+Apache Solr releases prior to 7.4 (i.e. Solr 5, Solr 6, and Solr 7 through 7.3) use Log4J 1.2.17 which may be vulnerable for installations using non-default logging configurations that include the JMS Appender, see <https://github.com/apache/logging-log4j2/pull/608#issuecomment-990494126> for discussion.
 
-The Prometheus Exporter Contrib is similarly separately affected.
+Solr's Prometheus Exporter uses Log4J as well but it does not log user input or data, so we don't see a risk there.
```

Comment:
Probably fine. The fact that log4j team released 2.16 to completely disable jndi hints at an uncertainty whether there are other ways of triggering dangerous code - but for now I think it's correct not to distract users with the exporter.